app/templates/base.jade

```diff
@@ -86,7 +87,7 @@ block footer
       if me.isAdmin()
         a(href='/admin') Admin
 
-      if usesSocialMedia && me.useSocialSignOn()
+      if usesSocialMedia && me.useSocialSignOn() && !features.china
```

Comment:
Should `features.china` be added to the `useSocialSignOn` method?
I am not sure you need this addition as `useSocialSignOn` is the inverse of `features.chinaUx`, which is correctly set on koudashijie?